webapp/geek_text/urls.py

```diff
@@ -17,11 +17,12 @@
 from django.conf.urls import url, include
 from django.contrib import admin
 from django.contrib.auth import views as auth_views
+
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^login/$', auth_views.login , name='login'),
+    url(r'^logout/$', auth_views.LogoutView.as_view(template_name='browse.html') , name='logout'),
```

Comment:
I'm confused what this line does... why does the logout url use the browse.html? Is this just a redirect to the browse page after the user logs out?

Reply:
That's exactly what it does. After the user is logged out that is the template that it renders.